doc/setup.md

```diff
@@ -19,6 +19,8 @@ symfony new . 2.8
 composer require victoire/victoire \
     friendsofsymfony/user-bundle:~2.0@dev \
     doctrine/orm \
+    doctrine/doctrine-fixtures-bundle \
+    symfony/assetic-bundle \
```

Comment:
Yes I was wrong about the symptom, the problem is that `AsseticBundle` is not mentioned in `AppKernel.php`